Source/WebKit/WebProcess/WebPage/AcceleratedSurface.h

```diff
@@ -61,6 +62,9 @@ class AcceleratedSurface {
     virtual void willRenderFrame() { }
     virtual void didRenderFrame(WebCore::Region&&) { }
 
+    virtual void damageRenderTargets(const WebCore::Damage&) { };
+    virtual const WebCore::Damage& renderTargetDamage() { ASSERT_NOT_REACHED(); return WebCore::Damage::invalid(); };
```

Comment:
RenderTarget is an implementation detail specific to the DMABuf surface, I think these should not use it. What about just `addDamage()` and `damage()` ?

Reply:
Also, I guess the old api will hit this assert if the setting is enabled, I think it should be pure virtual and implemented in old api, or remove the assert.